charts/vela-core/values.yaml

```
@@ -4,6 +4,8 @@
 
 replicaCount: 1
 useWebhook: true
+# By default, don't disable any builtin capabilities
+disableCaps: none
```

```suggestion
disableCaps: ''
```

不要 none， 空就代表没有。甚至不要这个参数

---

这是 values.yaml，要么写、要么不写，不存在不要。因为 build 出来的 chart，不推荐用户去删减 values.yaml。

文档里这里会有三类合法值
- ""/none
- all
- "autoscale,route"

改正 “” 我接受的。谢谢。